packages/testing/src/protos/google/protobuf/empty.proto

```diff
@@ -0,0 +1,52 @@
+// Protocol Buffers - Google's data interchange format
```

Comment:
can we include where this came from (somewhere, doesn't have to be in this file)

Reply:
I'll add a README